server/application/application_test.go

```diff
@@ -488,6 +488,44 @@ func TestCreateAppWithDestName(t *testing.T) {
 	assert.Equal(t, app.Spec.Destination.Server, "https://cluster-api.com")
 }
 
+func TestCreateAppShouldTruncateLabelIfTheLabelLengthIsGreaterThanSpecifiedLength(t *testing.T) {
```

Comment:
func name can be shorter (Ex: TestCreateAppWithInvalidNameLength). Test scenario can be described in the func documentation. Since setLongName func contains only one line, a separate func is not required.